lib/dotloop/loop.rb

```diff
@@ -42,6 +53,14 @@ def detail(profile_id:, loop_view_id:)
       Dotloop::Models::LoopDetail.new(loop_detail)
     end
 
+    def self.statuses
+      STATUS_MAP.keys
+    end
+
+    def statuses
+      self.class.statuses
+    end
```

Comment:
The hash is already exposed so it seems kind of like syntactic sugar we don't yet need.

I'm easy though.